packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -75,36 +72,53 @@ TIP: If your log file is part of your Salesforce DX project, you don’t need to
 
 ## Debug Your Code
 
-Set breakpoints, then replay your debug log and inspect your variables’ values.
+Set breakpoints or checkpoints, then replay your debug log and inspect your variables’ values.
 
 1. To set line breakpoints, open a `.cls` or `.trigger` file and click the column to the left of the line numbers.
+1. For more information than line breakpoints provide, add checkpoints. You can set up to five checkpoints to get heap dumps when lines of code run. All local variables, static variables, and trigger context variables have better information at checkpoints. Trigger context variables don’t exist in logs and are available only at checkpoint locations.  
```

Comment:
Minor issue: the heap dumps from checkpoints are taken _before_ the line of code they're on is executed.

Reply:
It may be worth mentioning that checkpoints will function like breakpoints while replay-debugging from the log that Checkpoints need to be set before the Apex is run and the log is generated.